docs/index.md

```diff
@@ -31,15 +31,15 @@ The project might not need a roof.
 
 Now the team can decorate and customise for their client.
 
-## How have we produced Stacks
+## How have we produced Stacks?
```

Comment:
This isn't valid markdown heading, hence why the listing failed. Either we make a statement, eg. How Stacks was produced or we turn off that listing rule for this file?